modules/aioseop_sitemap.php

```diff
@@ -3805,9 +4222,27 @@ function get_all_post_type_data( $args ) {
 					}
 				}
 			}
+
+			/**
```

Comment:
Could just use a reference for these.

```
/**
 * Filter to update the posts found.
 *
 * @since ?
 *
 * @param array $posts {
 *      Array of posts.
 *
 *      @type WP_Post $post {
 *          The post object.
 *
 *          @see WP_Post
 *          @link https://codex.wordpress.org/Class_Reference/WP_Post
 *      }
 * @param array $args {
 *     Arguments to send to WP_Query.
 *
 *     @see Parameter $args for WP_Query.
 *     @link https://codex.wordpress.org/Class_Reference/WP_Query#Parameters
 * }
 */
```